src/encoded/tests/datafixtures.py

```diff
@@ -470,10 +469,28 @@ def construct(testapp, lab, award, target, source, target_control):
         'lab': lab['@id'],
         'source': source['@id'],
         'construct_type': 'fusion protein',
-        'tags': [{'name': 'eGFP', 'location': 'C-terminal'}],
```

Comment:
just FYI it's common python practice (NOT JSON OR JS) to leave a trailing comma in a list or dict definition because it makes cutting and pasting easier.

Reply:
thanks, was not aware of that